<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageSettingsBase</class>
 <widget class="QDialog" name="PageSettingsBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Page Settings</string>
  </property>
  <layout class="QGridLayout">
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="previewGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Preview</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <layout class="QVBoxLayout" name="leftLayout" stretch="0,0,0">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="sizeGroup">
       <property name="title">
        <string>Page Size</string>
       </property>
       <layout class="QGridLayout">
        <item row="5" column="1">
         <widget class="QDoubleSpinBox" name="pageWidth">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <double>9999.989999999999782</double>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="heightLabel">
          <property name="text">
           <string>Height:</string>
          </property>
          <property name="buddy">
           <cstring>pageHeight</cstring>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QRadioButton" name="landscape">
          <property name="text">
           <string>Landscape</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QRadioButton" name="portrait">
          <property name="text">
           <string>Portrait</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QDoubleSpinBox" name="pageHeight">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <double>9999.989999999999782</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QComboBox" name="sizesList">
          <property name="toolTip">
           <string>Page Size</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QComboBox" name="typesList">
          <property name="toolTip">
           <string>Paper Type</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="twosided">
          <property name="text">
           <string>Two sided</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="widthLabel">
          <property name="text">
           <string>Width:</string>
          </property>
          <property name="buddy">
           <cstring>pageWidth</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="Line" name="line">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="scalingGroup">
       <property name="title">
        <string>Scaling</string>
       </property>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="spLabel">
          <property name="toolTip">
           <string>Distance between two lines on a standard 5-line staff</string>
          </property>
          <property name="whatsThis">
           <string>Distance between two lines on a standard 5-line staff</string>
          </property>
          <property name="text">
           <string>Staff space (sp):</string>
          </property>
          <property name="buddy">
           <cstring>staffSpace</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="staffSpace">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>0.001000000000000</double>
          </property>
          <property name="maximum">
           <double>999.999000000000024</double>
          </property>
          <property name="singleStep">
           <double>0.200000000000000</double>
          </property>
          <property name="value">
           <double>2.001000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="unitsGroup">
       <property name="title">
        <string>Units</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_1">
        <item row="0" column="1">
         <widget class="QComboBox" name="unitsList">
          <property name="currentText">
           <string/>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContents</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="unitsLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Unit of Measure:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="buttonLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="resetToDefault">
       <property name="text">
        <string>Reset to Default</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyToParts">
       <property name="text">
        <string>Apply to All Parts</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="rightLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="oddGroup">
       <property name="title">
        <string>Odd Page Margins</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0" colspan="2">
         <widget class="QDoubleSpinBox" name="oddLeftMargin">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <double>2000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="3" colspan="2">
         <widget class="QDoubleSpinBox" name="oddRightMargin">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <double>2000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="3">
         <widget class="QDoubleSpinBox" name="oddBottomMargin">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <double>2000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <spacer name="oddSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1" colspan="3">
         <widget class="QDoubleSpinBox" name="oddTopMargin">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <double>2000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="evenGroup">
       <property name="title">
        <string>Even Page Margins</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="3" colspan="2">
         <widget class="QDoubleSpinBox" name="evenRightMargin">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <double>2000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="3">
         <widget class="QDoubleSpinBox" name="evenBottomMargin">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <double>2000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QDoubleSpinBox" name="evenLeftMargin">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <double>2000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <spacer name="evenSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1" colspan="3">
         <widget class="QDoubleSpinBox" name="evenTopMargin">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <double>2000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="offsetLayout">
       <item>
        <widget class="QLabel" name="offsetLabel">
         <property name="text">
          <string>First page number:</string>
         </property>
         <property name="buddy">
          <cstring>pageOffset</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="pageOffset">
         <property name="minimum">
          <number>-99</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>sizesList</tabstop>
  <tabstop>pageWidth</tabstop>
  <tabstop>pageHeight</tabstop>
  <tabstop>portrait</tabstop>
  <tabstop>landscape</tabstop>
  <tabstop>staffSpace</tabstop>
  <tabstop>oddTopMargin</tabstop>
  <tabstop>oddLeftMargin</tabstop>
  <tabstop>oddRightMargin</tabstop>
  <tabstop>oddBottomMargin</tabstop>
  <tabstop>evenTopMargin</tabstop>
  <tabstop>evenLeftMargin</tabstop>
  <tabstop>evenRightMargin</tabstop>
  <tabstop>evenBottomMargin</tabstop>
  <tabstop>pageOffset</tabstop>
  <tabstop>applyToParts</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
